astroquery/ukidss/core.py

```diff
@@ -93,686 +66,5 @@ def __init__(self, username=None, password=None, community=None,
         else:
             self.login(username, password, community)
 
-    def _login(self, username, password, community):
```

Comment:
some of the docstrings needs to be updated from the inherited ones (e.g. ukidss specific filters, and fram_types). I'm not sure though what's the best approach to to that.